components/core/src/fs.rs

```diff
@@ -271,23 +271,14 @@ where
 /// # Failures
 ///
 /// * The path entries metadata cannot be loaded
-pub fn find_command_in_pkg<T, U>(
-    command: T,
-    pkg_install: &PackageInstall,
-    fs_root_path: U,
```

Comment:
An `fs_root_path` is required here when being called by FS_ROOT-aware logic such as binlinking, installing, etc. In the vast majority of the time, this will be a default `/`, but not always.

Reply:
The FS_ROOT here was used to resolve the paths fro `pkg_install.paths()`. Another change in this PR was to have `pkg_install.paths()` resolve the root from the FS_ROOT that pkg_install was constructed with. Assuming the pkg_install was constructed with the necessary FS_ROOT, passing it into this function should not be necessary. I must have missed a some place where that should have been passed into the costructor.  I was also operating under the incorrect assumption that FS_ROOT was only used for testing scenarios on linux.